pygef/plot_utils.py

```diff
@@ -137,26 +138,46 @@ def assign_color(df, classification, colors):
     """
     if colors is None:
         if classification == "robertson":
+            df = (
+                df.lazy()
```

Comment:
in case of a `df.lazy().single_op().collect()`, you can just write `df.single_op()`